plugins/hydra_nevergrad_sweeper/setup.py

```diff
@@ -6,7 +6,7 @@
     LONG_DESC = fh.read()
     setup(
         name="hydra-nevergrad-sweeper",
-        version="1.0.0rc2",
+        version="1.0.0rc4",
```

Comment:
why the rc version increments by 2?

Reply:
I am keeping the repo version on the next version, not on the released version.